app/javascript/packs/reducers/answers.js

```diff
@@ -16,19 +18,41 @@ function beforeFetchAnswer(state, action) {
     [answer.question_id]: {
       ...answer,
       saved: false,
```

Comment:
Where do we use this `saved` flag? Is there any way that we can use the fetch status constants instead?